dGame/Character.h

```diff
@@ -309,12 +309,11 @@ class Character {
 	const int64_t GetCoins() const { return m_Coins; }
 
     /**
-     * Updates the current amount of coins of the character by a specified amount, for achievements this is not sent
-     * as it's tracked by the client
+     * Updates the current amount of coins of the character by a specified amount
      * @param newCoins the amount of coins to update by
-     * @param message whether to notify the client of the change
+     * @param coinSource The source of the loot
      */
-    void SetCoins(int64_t newCoins, bool message = true);
+    void SetCoins(int64_t newCoins, int32_t coinSource);
```

Comment:
```suggestion
    void SetCoins(int64_t newCoins, eLootSourceType coinSource);
```

Reply:
ahh I understand.  since eLootSourceType is defined in dCommonVars.h as a int32_t, i can use eLootSourceType as the variable type since its basically a nickname.  Is this the correct understanding or no?

Reply:
Yeah basically. With the added benefit that the compiler warns you when you try to set any integer to it that isn't annotated the same way.